src/object/deduplicate_key.c

```diff
@@ -0,0 +1,436 @@
+/*
+ * Copyright 2008 Search Solution Corporation
```

Comment:
This is a new file, only "Copyright 2016 CUBRID Corporation" should exist.

Reply:
Please check the other files too.

Reply:
I will fix the deduplicate_key.h and deduplicate_key.c files.